README.md

```diff
@@ -23,7 +23,14 @@ git clone https://github.com/Robinlovelace/spatial-microsim-book.git
 Then navigate into the downloaded folder and open the `spatial-microsim-book.Rproj` RStudio project file. The [build.R](https://github.com/Robinlovelace/spatial-microsim-book/blob/master/build.R) file should contain everything you need to build the book, although you'll need to have installed a number of packages. These can be installed with the following command:
 
 ```
-install.packages("knitr", "rmarkdown", "png", "ggmap", "dplyr", "ipfp", "rgeos", "mipfp", "rgdal", "gridExtra") 
+pkgs <- c("knitr", "rmarkdown", "png", "ggmap", "dplyr", "ipfp", "rgeos", "mipfp", "rgdal", "gridExtra", "tidyr", "mlogit")
+install.packages(pkgs)
+```
+
+To install `ipfp` on Ubuntu (tested with version 14.04, the latest 'long-term support' version at the time of writing) a number of additional packages are required. These can be installed from the command line with:
+
+```
+sudo apt-get install liblapack-dev liblapack3 libopenblas-base libopenblas-dev
```

Comment:
Actually I've just double-checked the requirements and the *blas packages aren't required. I'm amend for a future pull request. The liblapack* packages are needed though...